docs/thegraph-connectors.md

```diff
@@ -1,26 +1,54 @@
-# TheGraph template for Aragon subgraphs
+# TheGraph connectors
+
+This section contains information about how our TheGraph connectors work, and how to use one of the existing connectors as a template for your own Aragon app connector.
+
+All existing connectors target all instances of a specific app, and can be deployed on any network. However, they can be modified to connect to any other Aragon app. In theory, it can even be used to connect to anything Aragon related, not just apps.
+
+## Architecture
+
+The connectors are composed of a root project which implements the base TheGraph connector, and an underlaying subgraph. Both components are specified below.
+
+### The connector
+
+The "src" folder contains the TheGraph connector wrapper which simply uses "@aragon/connect-thegraph" to perform GraphQL queries, parse them, and present them in the form of entity objects consistent with the "@aragon/connect" API.
+
+Use "index.ts" to pick which entities and objects are exposed to other packages.
+
+In "connect.ts", add functions that the entities of your connector will use. For example, the Vote entity will call its "castsForVote(...): Promise<Cast[]>" function. These functions all follow the same structure of (1) performing a query, (2) parsing the results of a query, and (3) wrapping and returning the results in the appropriate entity.
+
+Each of the steps described above is separated in the "entities", "parsers", and "queries" folders for clarity.
+
+Notice the file "queries/types.ts". This file is generated by a 3rd party tool, and provides Typescript types for GraphQL results. It is completely optional, but can be super handy when writing parsers.
```

Comment:
I think we should specify which tool is creating this `types.ts` file.

Reply:
This should be changed soon. I'm working in a way to automate things. For now, there's comments in the types files.